src/cache_cf.cc

```diff
@@ -620,18 +628,28 @@ parseConfigFileOrThrow(const char *file_name)
     return err_count;
 }
 
-// TODO: Refactor main.cc to centrally handle (and report) all exceptions.
-int
+void
 parseConfigFile(const char *file_name)
 {
+    static const char *kparseBlurb = "Log lines above or 'squid -k parse' may have more details.";
```

Comment:
Please inline this. There is no need to store this literal in a variable, especially a static one.